roles/common/tasks/subscription_manager.yml

```diff
@@ -0,0 +1,10 @@
+# subscribe a RHEL7 system to RHN
+
+- name: register to RHN
+  shell: subscription-manager register --username={{rhn-user}} --password={{rhn-pass}}
```

Comment:
@jdextercsc Just wondering why you didn't use the [subscription manager](http://docs.ansible.com/redhat_subscription_module.html) module.

And the use of shell vs command (http://docs.ansible.com/shell_module.html)